crates/oct-cloud/Cargo.toml

```diff
@@ -12,4 +12,6 @@ aws-sdk-iam = { workspace = true }
 base64 = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 mockall = { workspace = true }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
```

Comment:
You need to add these dependencies to the root Cargo.toml. And use them here with `{ workspace = true }` construction